File: Desenvolvimento Back-End PHP/03/poo/ContaPoupanca.php
<?php
    class ContaPoupanca extends Conta {
        
        private int $_diaDeposito;

        function setDiaDeposito(int $diaDeposito)
        {
            $multiploCinco = $diaDeposito % 5; 

            if ($diaDeposito < 30 && 
                ($multiploCinco == 0)) {
                $this->_diaDeposito = $diaDeposito;
            }
        }

        function VerLucro()
        {
        }

    }
?>

File: 05-engenharia-software/modelagem-sistemas/uml-poo/Exemplo Aula/Exemplo_conta_construct/src/ContaPoupanca.java

public class ContaPoupanca extends Conta {
	private int diaDeposito;
	
	public ContaPoupanca(String agencia, String conta, int diaDeposito) {
		super(agencia, conta);
		this.diaDeposito = diaDeposito;
	}

	
	public int getDiaDeposito() {
		return diaDeposito;
	}

	public void setDiaDeposito(int diaDeposito) {
		this.diaDeposito = diaDeposito;
	}

	public float verLucro() {
		return getSaldo();
	}
	
	/*public String toString() {
		return "Exemplos de saída";
	}*/
	
}
